pelican/themes/notmyidea/static/css/main.css

```diff
@@ -147,7 +147,8 @@ aside, nav, article, figure {
 }
 
 /***** Layout *****/
-.body {clear: both; margin: 0 auto; width: 800px;}
+.body {clear: both; margin: 0 auto; max-width: 800px;}
+img { max-width: 100%; }
```

Comment:
This (and another img max-width in percents) I do not understand.

Reply:
Thanks, that one is important because it prevents images from breaking out the side of the screen and making horizontal scrolling.

Reply:
I don't have any such images in my blog that aren't within `<figure>` elements, but it's it's a fairly standard setting in responsive CSS.